docs/hub/model-cards-components.md

```diff
@@ -0,0 +1,39 @@
+# Model Card components
+
+Model cards supports additionnal components you can place in your markdown. 
```

Comment:
```suggestion
**Model Card Components** are special elements that you can inject directly into your Model Card markdown to display powerful interface widgets directly in your model page. These components are authored by us, feel free to share ideas about new Model Card component in [this discussion](https://huggingface.co/spaces/huggingchat/chat-ui/discussions/312).
```

Reply:
not sure i'd say "widgets", might be confusing with the main widgets

"components" is ok no?